### PR TITLE
Migrate set-secret tool to actions framework

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Review by isoos — APPROVED** [review]

File: app/lib/admin/actions/secret.dart [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  summary: 'View, set or clear a secret value',
+  description: '''
+Get, set or remove a "secret" admin value affecting the site.
+
+If a `--value` is given, the secret will be set to that value. An empty value
+will clear the secret.
+
+To just inspect a secret, use the `get-secret` action.
+
+Possible secrets are [${SecretKey.values}] or prefixed ${SecretKey.oauthPrefix}
```
Comment (szakarias) [inline app/lib/admin/actions/secret.dart:19]: nit: add period at the end.

File: app/lib/admin/actions/secret.dart [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+The `announcement` secret will update the banner shown at top of pub.dev site
```
Comment (szakarias) [inline app/lib/admin/actions/secret.dart:21]: Perhaps: at top of -> at the top of the

**Review by szakarias — APPROVED** [review]
LGTM

File: app/lib/admin/actions/secret.dart (hunk 2 above)
Comment (sigurdm) [reply app/lib/admin/actions/secret.dart:21]: Oh - missed this. Will make follow up

**Pull request merged**